tools/Performance/DynamoPerformanceTests/PerformanceTestConsoleApp.cs

```diff
@@ -1,48 +1,120 @@
 ﻿using System;
 using System.IO;
+using BenchmarkDotNet.Configs;
 using BenchmarkDotNet.Running;
+using NDesk.Options;
 
 namespace DynamoPerformanceTests
 {
     public class Program
     {
-        /// <summary>
-        /// This is the entrance of Dynamo Performance Console App
-        /// </summary>
-        /// <param name="args"></param>
+        private enum Command
+        {
+            Benchmark,
+            Debug,
+            InProcess,
```

Comment:
@DynamoDS/dynamo Looking for insights on the naming, I am pretty sure people will be confused right away. Shall we consider: 
```
 private enum Command
        {
            Benchmark,
            NonOptimizedBenchmark,
            DebugInProcess,
            Compare,
}
```